From 6f643d2761650e53fd3b647526110a8473138b5d Mon Sep 17 00:00:00 2001
From: Thomas Haller <thaller@redhat.com>
Date: Thu, 8 Jan 2015 11:55:50 +0100
Subject: core: add /bin to the default search path

https://bugzilla.gnome.org/show_bug.cgi?id=742576
(cherry picked from commit abbca50ca9d552a28eca792f12ec618df6128fff)

diff --git a/src/NetworkManagerUtils.c b/src/NetworkManagerUtils.c
index 2c291cd..6747b65 100644
--- a/src/NetworkManagerUtils.c
+++ b/src/NetworkManagerUtils.c
@@ -814,6 +814,7 @@ const char *const NM_PATHS_DEFAULT[] = {
 	"/sbin/",
 	"/usr/sbin/",
 	"/usr/local/sbin/",
+	"/bin/",
 	"/usr/bin/",
 	"/usr/local/bin/",
 	NULL,
-- 
cgit v0.10.2

